cmd/monaco/integrationtest/v2/supportarchive_e2e_test.go

```diff
@@ -0,0 +1,178 @@
+//go:build integration
+
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2
+
+import (
+	"archive/zip"
+	"bytes"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/log"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/testutils"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/timeutils"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/trafficlogs"
+	"github.com/stretchr/testify/assert"
+	"io"
+	"path/filepath"
+	"testing"
+
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/runner"
+	"github.com/spf13/afero"
+)
+
+func TestSupportArchiveIsCreatedAsExpected(t *testing.T) {
+	configFolder := "test-resources/support-archive/"
+	manifest := configFolder + "manifest.yaml"
+	fixedTime := timeutils.TimeAnchor().Format(trafficlogs.TrafficLogFilePrefixFormat) // freeze time to ensure log files are created with expected names
+
+	RunIntegrationWithCleanup(t, configFolder, manifest, "valid_env", "SupportArchive", func(fs afero.Fs, _ TestContext) {
+		cleanupLogsDir(t)
```

Comment:
move to "RunIntegrationWithCleanup" ?

Reply:
it's not really an issue that any other test has, we write some unnecessary files into the container running the test, but no other test cares about them being there. 

If it was a general issue (and looked easy) I'd have refactored the traffic logger to accept a filesystem on creation, instead of always using the OS filesystem.

Reply:
also having this in `RunIntegrationWithCleanup` would not help for the error cases, as that util assumes that manifest can actually be loaded and it can connect to the environment. So it'd need to be called explicitly for these tests still.